src/Modules/OrchardCore.Commerce/Startup.cs

```diff
@@ -212,6 +216,15 @@ public override void ConfigureServices(IServiceCollection services)
     }
 }
 
+public class FallBackPriceStartup : StartupBase
```

Comment:
Since the word is "fallback".

```suggestion
public class FallbackPriceStartup : StartupBase
```